app/admin/contacts.rb

```diff
@@ -0,0 +1,3 @@
+ActiveAdmin.register Contact do
+  actions :all, except: %i[update create new edit]
+end
```

Comment:
For the future, it's better if you specify the things you want in the admin page. This way, you can have more control.
For example, if a new "secret" attribute is added, it will be automatically added to the admin page, but maybe you don't want that attribute in the admin dashboard.